lib/cadet/accounts/form/login.ex

```diff
@@ -1,4 +1,7 @@
 defmodule Cadet.Accounts.Form.Login do
+  @moduledoc """
+  Validates login information provided through forms.
```

Comment:
This module (and the other form module) exist not only as validation but to implement the form entity itself, the validation is just something done for Ecto entities.
A better documentation would be to give a short description of what this entity represent (a login form comprises of e-mail and password)